examples/dockerdev/docker-compose.yml

```diff
@@ -27,13 +27,13 @@ x-backend: &backend
     - .:/app:cached
     # or for selinux: https://docs.docker.com/storage/bind-mounts/#configure-the-selinux-label
     # - .:/app:cached,z
-    - rails_cache:/app/tmp/cache
-    - bundle:/usr/local/bundle
-    - node_modules:/app/node_modules
-    - packs:/app/public/packs
-    - packs-test:/app/public/packs-test
-    - .dockerdev/.psqlrc:/root/.psqlrc:ro
-    - .dockerdev/.bashrc:/root/.bashrc:ro
+    - rails_cache:/app/tmp/cache:delegated
+    - bundle:/usr/local/bundle:delegated
+    - node_modules:/app/node_modules:delegated
+    - packs:/app/public/packs:delegated
+    - packs-test:/app/public/packs-test:delegated
+    - .dockerdev/.psqlrc:/root/.psqlrc:ro,cached
+    - .dockerdev/.bashrc:/root/.bashrc:ro,cached
```

Comment:
I wonder if `log` shouldn't be listed here too?

Reply:
Yes, makes sense. Added. Thanks!